<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sunseeker CAN USB</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Pictures/wmu_sunseeker.jpg</normaloff>../Pictures/wmu_sunseeker.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="lblbxMessage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>771</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Message</string>
    </property>
    <widget class="QPushButton" name="btnClearData">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear Data</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>120</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Msg Id</string>
     </property>
     <widget class="QLineEdit" name="txtMsgId">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>101</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Flags</string>
     </property>
     <widget class="QCheckBox" name="chkExtended">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>97</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Extended</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkRTR">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>RTR</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>60</y>
       <width>411</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Data</string>
     </property>
     <widget class="QLineEdit" name="txtData7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>41</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>0x00</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtData6">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>41</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>0x00</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtData5">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>41</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>0x00</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtData4">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>41</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>0x00</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtData3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>41</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>0x00</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtData2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>20</y>
        <width>41</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>0x00</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtData0">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>20</y>
        <width>41</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>0x00</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtData1">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>20</y>
        <width>41</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>0x00</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>411</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>All values can be given in the decimal or hexidecimal for(0x...)</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>140</y>
      <width>321</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Adapters</string>
    </property>
    <widget class="QComboBox" name="cmbAdaptSelect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>141</width>
       <height>27</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAdaptSelect">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>31</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnGetAdapters">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Get Adapters</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Available serial numbers</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Serial # to open</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtSreialOpen_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>70</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnSendMsg">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>140</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Message</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSendBurst">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>170</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Burst</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtBurstSize">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>170</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>781</width>
      <height>351</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="allMsg">
     <attribute name="title">
      <string>All Messages</string>
     </attribute>
     <widget class="QTableWidget" name="txtMsgs">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>1</y>
        <width>771</width>
        <height>291</height>
       </rect>
      </property>
      <property name="baseSize">
       <size>
        <width>4</width>
        <height>20</height>
       </size>
      </property>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Timestamp</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>High Info</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>High Data</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Low Info</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Low Data</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>660</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Sent frames:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>680</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Recieved frames:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSentFrames">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>660</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0000000</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblRecievedFrames">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>680</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0000000</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClrMsgList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>660</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Message List</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExit">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>660</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>210</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>New Msg Tab</string>
    </property>
    <widget class="QPushButton" name="btnAddTab">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Tab</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>140</y>
      <width>231</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0" colspan="2">
      <widget class="QPushButton" name="btnOpenClose">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox_6">
       <property name="title">
        <string>Baudrate</string>
       </property>
       <widget class="QComboBox" name="cmbBaudrate">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>81</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>250 kbs</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1000 mbs</string>
         </property>
        </item>
       </widget>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>Flags</string>
       </property>
       <widget class="QCheckBox" name="chkTimeStamp">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Time Stamp</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="chkSlowMode">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>101</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Slow Mode</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>812</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Log"/>
    <addaction name="actionSave_Log_as"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Log">
   <property name="text">
    <string>Open Log</string>
   </property>
  </action>
  <action name="actionSave_Log_as">
   <property name="text">
    <string>Save Log as</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>txtMsgId</tabstop>
  <tabstop>chkExtended</tabstop>
  <tabstop>chkRTR</tabstop>
  <tabstop>txtData7</tabstop>
  <tabstop>txtData6</tabstop>
  <tabstop>txtData5</tabstop>
  <tabstop>txtData4</tabstop>
  <tabstop>txtData3</tabstop>
  <tabstop>txtData2</tabstop>
  <tabstop>txtData1</tabstop>
  <tabstop>txtData0</tabstop>
  <tabstop>btnClearData</tabstop>
  <tabstop>cmbAdaptSelect</tabstop>
  <tabstop>btnGetAdapters</tabstop>
  <tabstop>btnAdaptSelect</tabstop>
  <tabstop>txtSreialOpen_3</tabstop>
  <tabstop>cmbBaudrate</tabstop>
  <tabstop>btnOpenClose</tabstop>
  <tabstop>btnSendMsg</tabstop>
  <tabstop>btnSendBurst</tabstop>
  <tabstop>txtBurstSize</tabstop>
  <tabstop>chkTimeStamp</tabstop>
  <tabstop>chkSlowMode</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>btnClrMsgList</tabstop>
  <tabstop>btnExit</tabstop>
  <tabstop>btnAddTab</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
